.github/workflows/bump-bulk-cdk-and-release-connectors.yml

```diff
@@ -10,8 +10,82 @@ on:
         options:
           - airbytehq/airbyte
 jobs:
-  bump-cdk-version-and-merge:
+  publish-bulk-cdk:
+    uses: ./.github/workflows/publish-bulk-cdk.yml
+    secrets: inherit
+  bump-cdk-version:
     runs-on: ubuntu-24.04
+    outputs:
+      modified_connectors: ${{ steps.export-connection-modified.outputs.modified_connectors }}
+      # Output the matrix config as a JSON string
+      modified_connectors_json: ${{ steps.export-connection-modified.outputs.modified_connectors_json }}
+    needs: [publish-bulk-cdk]
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
+          # Fetch all history for all tags and branches
+          fetch-depth: 0
+      - name: Fetch latest Bulk Load CDK version
+        id: fetch-version
+        run: |
+          # 1. Define the target URL
+          REPO_URL="https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/io/airbyte/bulk-cdk/bulk-cdk-core-load/"
+          echo "Fetching versions from $REPO_URL" >&2 # Log output to standard error
+
+          # 2. Fetch HTML, extract versions (X.Y), sort, get the latest
+          latest_version=$(curl -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' -s "$REPO_URL" | grep -o 'href="[0-9]\+\.[0-9]\+/"' | sed 's/^href="//; s/\/"$//' | sort -V | tail -n 1)
+          echo "Latest version found: $latest_version"
+
+          # 3. Set the output parameter for GitHub Actions
+          # This makes "latest_version" available to subsequent steps
+          echo "latest_version=$latest_version" >> "$GITHUB_OUTPUT"
+      - name: Configure Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
we miiight run into the same problem as ryan did here https://airbytehq-team.slack.com/archives/C07GM7XHB9A/p1746049280861499

(i.e. if we push a commit as github-actions, the PR might get blocked b/c of CLA nonsense. Probably solvable somehow?)

Reply:
I changed it to octavia